pkgs/misc/drivers/epson-201202w/default.nix

```diff
@@ -0,0 +1,74 @@
+{ lib, stdenv, fetchurl, rpmextract, autoreconfHook, file, libjpeg, cups }:
+
+  stdenv.mkDerivation rec {
+    version = "1.0.0";
+    filterVersion = "1.0.0";
```

Comment:
this could affect the derivation, since it is provided as input to to `stdenv.mkDerivation`, would you rather just use `version` or move this into a let block?
```nix
let
  filterVersion = "1.0.0";
in stdenv.mkderivation rec {
```
```

Reply:
Let's combine them